scripts/exgfs_wave_prep.sh

```diff
@@ -249,7 +262,8 @@
         sed "s/^/wave_prnc_ice.out : /g" wave_prnc_ice.out
         echo ' '
         set_trace
```

Comment:
Do we need these `set_trace`s in any of these scripts? If not, please remove.

Reply:
we don't need the `set_trace` stuff if we tighten up this and other scripts and have a consistent logging/messaging.